Src/Base/AMReX_ParmParse.cpp

```diff
@@ -538,6 +538,11 @@ read_file (const char* fname, std::list<ParmParse::PP_entry>& tab)
     //
     if ( fname != nullptr && fname[0] != 0 )
     {
+#ifdef AMREX_USE_MPI
+        if (!amrex::Initialized())
+            amrex::Error("read_file: AMReX must be initialized");
+#endif
+
         Vector<char> fileCharPtr;
         std::string filename = fname;
         ParallelDescriptor::ReadAndBcastFile(filename, fileCharPtr);
```

Comment:
The `ParallelDescriptor::ReadAndBcastFile` with MPI needs `amrex::Initialized()` but should work if build w/o MPI.